src/service/basket.test.js

```diff
@@ -4,43 +4,56 @@ const mockProduct = { name:'nikey', price: 1255, quantity: 1 };
 const product2 = { name:'cream', price: 342, quantity: 1 };
 
 
-test('that init basket can be returned', () => {
-  expect(basket.get().products.length).toEqual(0);
-  expect(basket.get().total).toEqual(0);
+beforeEach(() => {
```

Comment:
this can be inside the suite so we know it applies to it, in case we have more suites in the future.
basket.clear() does not need to be returned since it does not return any value.